substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/elf/dwarf/DwarfSectionImpl.java

```diff
@@ -55,7 +61,13 @@
 import java.util.Set;
 import java.util.stream.Stream;
 
-import static com.oracle.objectfile.elf.dwarf.DwarfDebugInfo.DW_OP_stack_value;
+import static com.oracle.objectfile.elf.dwarf.constants.DwarfSectionName.DW_ABBREV_SECTION;
```

Comment:
Instead of importing individual enum values use a single
```
import com.oracle.objectfile.elf.dwarf.constants.DwarfSectionName
```

Reply:
If you look further up the file you will see that I already have a single

    import com.oracle.objectfile.elf.dwarf.constants.DwarfSectionName

The reason I am using the static imports is that this avoids the need to have every mention of a constant qualified by the enum it belongs to e.g. at line 285 the current code is

        markRelocationSite(pos, ObjectFile.RelocationKind.DIRECT_8, TEXT_SECTION.value(), l);

If I remove the static import I need to change this to

        markRelocationSite(pos, ObjectFile.RelocationKind.DIRECT_8, DwarfSectionName.TEXT_SECTION.value(), l);

So, your suggestion that I remove the static imports implies that I would also need to change every such mention of an enum value so that it is qualified with the name of the enum class it belongs to.

Is that really what you want?

Reply:
> So, your suggestion that I remove the static imports implies that I would also need to change every such mention of an enum value so that it is qualified with the name of the enum class it belongs to.

Yes, that's completely fine. BTW, intellij can do that automatically for you.

![image](https://github.com/oracle/graal/assets/16901644/ec5285b7-4f58-4c6a-b077-5bafb3f665ac)

Reply:
Yes, I understand that this can be done automatically. I wanted to be sure this is what you want. Personally, I would *much rather* avoid having to include the enum class qualifiers as:

1. it adds needless clutter to the code (readability--)
2. that clutter clouds the otherwise obvious and immediate implication that that these symbols are precisely those defined by the DWARF spec (meaningfulness--)
3. qualification of the names will not automatically result in a compile failure should someone by any chance attempt to reuse the DWARF spec names (reliability--).

However, if you are sure you really prefer it this way then I will change it.

Reply:
It's a tradeoff. Having a huge block of `static imports` is also not pretty. And every time you are using a new enum value another one is added. There is a tipping point where I also opt for static imports over qualified access. E.g. if there are many many places where I would need that one single enum value. But this is not really the case here so I do in fact prefer a import + qualified access wherever a value is used.